BuildAndTest.proj

```diff
@@ -79,17 +79,23 @@
           SkipUnchangedFiles="true"
           UseHardlinksIfPossible="true" />
 
-    <MakeDir Directories="$(CoreClrTestDirectory)\xUnitResults" />
+    <MakeDir Directories="$(CoreClrTestDirectory)/xUnitResults" />
+
+    <!-- Publish binaries for CoreClr test -->
+    <MSBuild BuildInParallel="true"
+             Projects="src/Test/DeployCoreClrTestRuntime/DeployCoreClrTestRuntime.csproj"
```

Comment:
The last commit said removing code from a bad merge yet I still see the `/` here.  Can we grep this file, ensure there are no `/` remaining and then push?  Prefer not to have another iteration on this file where we remove `/`.